CHANGES.md

```
@@ -21,10 +21,13 @@ To be released.
         they switch to `redirect_uris`.
      -  Expose `redirect_uri`, `redirect_uris`, and `scopes` to verify
         credentials for apps.
+ -  Added support for RFC 8414 for OAuth Authorization Server metadata endpoint.
```

Markdownlint

Expected: 0; Actual: 1
Unordered list indentation

(MD007, ul-indent)